org-code-javabuilder/lib/src/test/java/org/code/javabuilder/InputRedirectionStreamTest.java

```diff
@@ -0,0 +1,46 @@
+package org.code.javabuilder;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class InputRedirectionStreamTest {
+  private InputRedirectionStream stream;
+  private InputAdapter inputAdapter;
+
+  @BeforeEach
+  public void setup() {
```

Comment:
Heh, one of the major decisions that you have to make when writing the first unit tests -- is "setup" one word or two?  (It's spelled "setup" here but "setUp" above.)

Reply:
Haha ok. I standardized :)